src/olympia/bandwagon/forms.py

```diff
@@ -195,43 +196,49 @@ def clean_icon(self):
         icon = self.cleaned_data['icon']
         if not icon:
             return
-        if icon.content_type not in ('image/png', 'image/jpeg'):
+        icon_check = ImageCheck(icon)
+        if (icon.content_type not in ('image/png', 'image/jpeg') or
```

Comment:
On a second note… this should probably re-use `constants.IMG_TYPES` - which interestingly enough contains `image/jpg` but shouldn't imho

Reply:
Yeah I'll re-use that constant... Searching for `image/{something}` I also just noticed there is another place where we do image upload that would benefit from this... I'll fix it all